src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateRepresentedClaimCopyTest.java

```diff
@@ -27,6 +41,16 @@ public void shouldReturnPdfDocumentIfEverythingIsFine() throws Exception {
         given(pdfServiceClient.generateFromHtml(any(), any()))
             .willReturn(PDF_BYTES);
 
+        given(documentUploadClientApi.upload(anyString(), any(List.class)))
```

Comment:
This block (lines 44 - 53) is duplicated 3 times in this pull request. Please extract this to some test setup helper.